vnext/Shared/RuntimeOptions.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+
+namespace Microsoft::React {
+
+/// <summary>
+/// Sets a global boolean value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global key</param>
+void __cdecl SetRuntimeOptionBool(std::string &&name, bool value) noexcept;
+
+/// <summary>
+/// Sets a global signed integer value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global boolean key</param>
```

Comment:
><param name="name">Global boolean key</param> [](start = 4, length = 45)

Nit: doesn't seem to line up with actual parameters.

Reply:
True. I'll have to update it in master too.